packages/cli/src/controllers/e2e.controller.ts

```diff
@@ -29,23 +29,23 @@ if (!inE2ETests) {
 const tablesToTruncate = [
 	'auth_identity',
 	'auth_provider_sync_history',
-	'credentials_entity',
+	'credentials',
```

Comment:
```suggestion
	'credential',
```

Reply:
will update them, but considering that the tests did not fail makes me think truncating this table it's not neccesary